samples/samples/async_client_sample.py

```diff
@@ -0,0 +1,648 @@
+
+     """Creates batch sessions."""
+     response = await async_client.batch_create_sessions(request = request)
+
+     """Prints the sessions batchwise."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client batch_create_session]
+
+
+# [START async client create_session]
+async def create_session(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     """Request object for create session."""
+     request = CreateSessionRequest(
+         database = database_object.name,
+     )
+
+     """Creates a session."""
+     response = await async_client.create_session(request = request)
+
+     """Prints the session."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client create_session]
+
+
+async def main():
+    parser = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+    parser.add_argument("--instance_id", help="Your Cloud Spanner instance ID.")
+    parser.add_argument(
+        "--database_id", help="Your Cloud Spanner database ID.", default="example_db"
+    )
+    parser.add_argument(
+        "--session_name", help="session name."
+    )
+    parser.add_argument(
+        "--n", help="number of queries."
+    )
+    subparsers = parser.add_subparsers(dest="command")
+    subparsers.add_parser(
+        "create_session",
+        help=create_session.__doc__
+    )
+    subparsers.add_parser(
+        "batch_create_session",
+        help=batch_create_session.__doc__
+    )
+    subparsers.add_parser(
+        "get_session",
+        help=get_session.__doc__
+    )
+    subparsers.add_parser(
+        "list_sessions",
+        help=list_sessions.__doc__
+    )
+    subparsers.add_parser(
+        "delete_session",
+        help=delete_session.__doc__
+    )
+    subparsers.add_parser(
+        "execute_sql",
+        help=execute_sql.__doc__
+    )
+    subparsers.add_parser(
+        "execute_streaming_sql",
+        help=execute_streaming_sql.__doc__
+    )
+    subparsers.add_parser(
+        "execute_batch_dml",
+        help=execute_batch_dml.__doc__
+    )
+    subparsers.add_parser(
+        "read",
+        help=read.__doc__
+    )
+    subparsers.add_parser(
+        "streaming_read",
+        help=streaming_read.__doc__
+    )
+    subparsers.add_parser(
+        "begin_transaction",
+        help=begin_transaction.__doc__
+    )
+    subparsers.add_parser(
+        "commit",
+        help=commit.__doc__
+    )
+    subparsers.add_parser(
+        "rollback",
+        help=rollback.__doc__
+    )
+    subparsers.add_parser(
+        "partition_query",
+        help=partition_query.__doc__
+    )
+    subparsers.add_parser(
+        "partition_read",
+        help=partition_read.__doc__
+    )
+    args = parser.parse_args()
+
+    async_client, database_object = getObjects(args.instance_id, args.database_id)
+
+    if args.command == "create_session":
```

Comment:
As all your conditions are of the type x == value with x being the same in every condition, switch case usually makes sense.

Reply:
As python does not have an in-built switch case statement should I add a class to implement that?

Reply:
No, you need not implement it.